docs/schemes/local.md

```diff
@@ -83,3 +84,9 @@ This option can be used to disable all token handling. Useful for Cookie only fl
 - Default: `Bearer`
 
  Authorization header type to be used in axios requests.
+ 
+ ### `autoFetchUser`
```

Comment:
Would you please create a separate PR for this feature?

Reply:
@pi0 Sure!

Reply:
Revert commits: https://github.com/nuxt-community/auth-module/pull/541/commits/223a59b46d07d44d23673d4081c1ef1f4a0b7b76 and https://github.com/nuxt-community/auth-module/pull/541/commits/7d0bc40360fd2c2de2e1ecf793bbf7f7487d0155
autoFetchUser PR: https://github.com/nuxt-community/auth-module/pull/543